### DISCUSSION TITLE
How to get work as Solidity Developer without Prior Experience

### DISCUSSION THREAD

**Discussion by 0xkriswebchain** [Cyfrin/foundry-full-course-cu#1480] [General]
Hello everyone, I hope you guys are doing well. I want to share that recently I have been dived into the blockchain and smart contracts. I got inspired with the opportunity to create something new. I am new to this industry and IT. Before that I used to work as content writer. Currently after getting enough understanding on how to write code, debug, compile, test, and deploy secure smart contracts. I have developed some projects with the help of Patrick Collins courses. 

I am confused and uncleared on next thing to do, that is finding work for creating solidity smart contract as entry level developer. Before this, I never written code in other language except HTML. However, I have some understanding about JavaScript. I want future guidance from you guys, what shall I work on. How to get work as entry level solidity developer into this industry without prior corporate experience. 

If you could share links that provides opportunities to entry level blockchain developers to do work, earn experience, let me know: 

Thank you for your time, 

Best regards

**Comment by EngrPips** [comment]
Hey @Kris291099 , it's awesome to hear how much you have grown and learnt and congratulations on learning that much but now let's  talk reality. If you are looking and hoping to get job as an entry level solidity developer there is an high chance that you will never get a job because honestly in most case scenario no one is looking to employ a beginner to write a code that would be handling millions of dollars if not billions. What i would advice is start studying the most successful projects in the web3 (uniswap, aave, curve, makerdao e.t.c) and maybe even build minimal clone of them. doing this would help you grow immensely that no one would consider you a beginner developer and at that point you will stand a better chance.

**Reply by 0xkriswebchain** [reply]
Thanks @EngrPips. I am really grateful with your piece of advice you gave me. I am looking forward for right guidance at the moment and share my progress status with you. I will be working on creating a successful web3 Dapps in future. In addition to it, I would be grateful if you could share some key techniques, tools and patterns a real blockchain developer uses on daily basis. Even though I know the key tools, but it would be great to gain some quick insight from a real blockchain developer who performs different tasks on daily basis. I am looking forward to connecting with you in future, as I am serious for this profession. Best regards.

**Reply by EngrPips** [reply]
Hello @Kris291099 , if i would be honest Patrick touch every techniques, tools and pattern you need to know and have to be a successful blockchain developer.

**Reply by 0xkriswebchain** [reply]
Thanks @EngrPips. Yes but do I also I have to learn about hyperledger as it has not been talked about in the courses.

**Reply by EngrPips** [reply]
If you see the need to learn it then go learn it, Most of the learning is done personally and no course would ever teach you everything you need to know.

**Reply by 0xkriswebchain** [reply]
Yes exactly 💯
